WordPress/Classes/ViewRelated/Stats/Shared Views/GhostViews/StatsGhostTableViewRows.swift

```diff
@@ -43,6 +43,6 @@ struct StatsGhostChartImmutableRow: StatsRowGhostable {
 
 private enum GhostCellStyle {
     static let muriel = GhostStyle(beatDuration: TimeInterval(0.75),
-                                   beatStartColor: .neutral(shade: .shade0),
-                                   beatEndColor: .neutral(shade: .shade5))
+                                   beatStartColor: .neutral(.shade0),
```

Comment:
We should maybe think about defining these in the Muriel UIColor extension – `ghostStart` and `ghostEnd` or something.

Reply:
@nheagy made it already in [#12415](https://github.com/wordpress-mobile/WordPress-iOS/pull/12415) 😊
I'm going to update this as soon `13.2` will be merged in `develop`. I added a reminder in the Master Issue.